src/components/Modals/Metadata/Metadata.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { Box, DataTable, Image, Text } from 'grommet'
+import { Close } from 'grommet-icons'
+import styled from 'styled-components'
+import { PlainButton } from '@zooniverse/react-components'
+import data from './mockData'
+import Satellite from 'images/satellite_map.png'
+import ConditionalLink from './components/ConditionalLink'
+
+const StyledDataTable = styled(DataTable)`
+  width: 100%;
+
+  thead { display: none; }
+  th { text-align: right; }
+`
+
+const Uppercase = styled(Text)`
+  text-transform: uppercase;
+`
```

Comment:
```suggestion
```

Shouldn't be necessary when using the `PlainButton`

Reply:
The label on a `PlainButton` is a [SpacedText](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-react-components/src/PlainButton/PlainButton.js#L23) component, which can take an `uppercase` prop to [transform](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-react-components/src/SpacedText/SpacedText.js#L9) all characters, but that prop is not being passed in from the PlainButton component.